### PR TITLE
Cobra flask respawn

### PR DESCRIPTION
**Pull request by marcosvf132**
On the previous change, the empty flask was added only on the map, but It suppouse to be on map and respawn every 8 hours. This PR add the respawn if the item is not on the position.
image

### CODE REVIEW

File: data/scripts/globalevents/cobra_flask.lua [hunk 1]
```diff
@@ -0,0 +1,14 @@
+local position = Position(33395, 32666, 5)
+
+local cobraEmptyFlask = GlobalEvent("Cobraflask")
+
+function cobraEmptyFlask.onThink(interval)
+local flask = Tile(positions):getItemById(36132)
```
Comment (dudantas) [inline data/scripts/globalevents/cobra_flask.lua]: The correct is "position"`and not "positions" 
But, you can remove the variable and put:
````local flask = Tile({x = 33395, y = 32666, z = 5}):getItemById(36132)````

**Review by dudantas — CHANGES_REQUESTED** [review]

File: data/scripts/globalevents/cobra_flask.lua (hunk 1 above)
Comment (marcosvf132) [reply data/scripts/globalevents/cobra_flask.lua]: ops

**Comment by attack787** [comment]
Don't work for me. I removed the item of map editor and used this script. Nothing happens

**Comment by marcosvf132** [comment]
Dont need to remove the item from your map, this script only respawn the item every 8 hours, so if you remove it, it will spawn again 8 hours later.

**Comment by attack787** [comment]
I reduced the time to test and this is working, thanks.

**Review by majestyotbr — APPROVED** [review]

**Pull request merged**